### DISCUSSION TITLE
The downloaded video is abnormal

### DISCUSSION THREAD

**Discussion by xin76** [aclap-dev/video-downloadhelper#1343] [Help and Q&A]
### Your question

The video played normally online, but after downloading, it was found that it could not be watched normally and there were serious problems with the picture.
![vlcsnap-2024-10-10-09h01m02s475](https://github.com/user-attachments/assets/34d26f9d-a852-4700-a8c1-90f1f7fd176d)
![vlcsnap-2024-10-10-09h01m06s164](https://github.com/user-attachments/assets/7b90eb60-65a1-4d37-b013-32917b94e5c7)

### Addon & CoApp version and details
Version 9.1.0.44 (stable), 
。

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I am just another user.

First off, you should never try to edit or abbreviate what you paste from the `Copy information to clipboard` data.  Post it all.  Alternatively, you could say something like this: Windows 11 Pro 64-bit version 24H2 build 26100.2033, Firefox 131.0.2 64-bit, licensed VDH 9.1.0.41 beta, CoApp 2.0.20.  There are more pieces of information that are usually needed that you have omitted.

But beyond that, your content is apparently protected by DRM.  Search this forum with `drm` as your search key.  There's plenty of information in here relevant to your problem.  Read some of it.

**Reply by 3ifish** [reply]
I have downloaded media that is garbled as shown in the OP, but a second or third try will provide a working version.  DRM is not necessarily the explanation.
